### ISSUE TITLE
Settings access is doing it wrong

### ISSUE THREAD

**Issue by chicagogrooves** [Differential/meteor-boilerplate#65]
The idea is that each setting has a key, and the correct settings file (or JSON) is passed in at process startup time, so your code would only contain

`Meteor.settings.public.meta.title`

not

`Meteor.settings[Meteor.settings.environment].public.meta.title`

It's seldom that production and dev settings ought to be in the same file anyway, since you don't want to check in prod settings to source control anyway.

**Comment by ryw** [comment]
@chicagogrooves i agree.
@nickw you agree too? you committed this version...

**Comment by nickw** [comment]
Yeah this was never fully implemented and was still in the experimentation phase. @chicagogrooves feel free to open a PR.

**Comment by chicagogrooves** [comment]
groovy, thanks! Sorry if that was a bit :trollface:
